From 54b1df85c66dd889a4e2721f5f31bfe229311f04 Mon Sep 17 00:00:00 2001
From: Manaswini Paluri <quic_mpaluri@quicinc.com>
Date: Mon, 9 Oct 2023 17:36:31 +0530
Subject: [PATCH] Add QCA vendor feature flag for TWT responder support in HT
 and VHT modes

Add a feature flag to indicate driver support for TWT responder for AP
operating in HT and VHT modes.

Signed-off-by: Manaswini Paluri<quic_mpaluri@quicinc.com>
---
 src/common/qca-vendor.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/common/qca-vendor.h b/src/common/qca-vendor.h
index 2a4086b76..7506140d5 100644
--- a/src/common/qca-vendor.h
+++ b/src/common/qca-vendor.h
@@ -1988,6 +1988,8 @@ enum qca_wlan_vendor_acs_hw_mode {
  *	with %QCA_WLAN_VENDOR_ATTR_CONFIG_AP_ALLOWED_FREQ_LIST.
  * @QCA_WLAN_VENDOR_FEATURE_ENHANCED_AUDIO_EXPERIENCE_OVER_WLAN: Flag indicates
  *	that the device supports enhanced audio experience over WLAN feature.
+ * @QCA_WLAN_VENDOR_FEATURE_HT_VHT_TWT_RESPONDER: Flag indicates that the device
+ *	in AP mode supports TWT responder mode in HT and VHT modes.
  * @NUM_QCA_WLAN_VENDOR_FEATURES: Number of assigned feature bits
  */
 enum qca_wlan_vendor_features {
@@ -2015,6 +2017,7 @@ enum qca_wlan_vendor_features {
 	QCA_WLAN_VENDOR_FEATURE_PROT_RANGE_NEGO_AND_MEASURE_AP = 21,
 	QCA_WLAN_VENDOR_FEATURE_AP_ALLOWED_FREQ_LIST = 22,
 	QCA_WLAN_VENDOR_FEATURE_ENHANCED_AUDIO_EXPERIENCE_OVER_WLAN = 23,
+	QCA_WLAN_VENDOR_FEATURE_HT_VHT_TWT_RESPONDER = 24,
 	NUM_QCA_WLAN_VENDOR_FEATURES /* keep last */
 };
 
-- 
2.43.0

